gc.c

```diff
@@ -798,6 +798,8 @@ typedef struct rb_objspace {
 	size_t total_freed_objects;
 	size_t total_allocated_pages;
 	size_t total_freed_pages;
+        size_t total_time_ns;
```

Comment:
Not sure how important is is, but if I'm not mistaken storing nanoseconds in `size_t` means that on 32bit systems the counter will overflow after 4.3 seconds spent in GC.